src/libs/portfolio/portfolio.ts

```diff
@@ -88,7 +89,29 @@ export class Portfolio {
     // Get hints (addresses to check on-chain) via Velcro
     const start = Date.now()
     const networkId = this.network.id
-    const hints = await this.batchedVelcroDiscovery({ networkId, accountAddr, baseCurrency })
+
+    // Make sure portfolio lib still works, even in the case Velcro discovery fails.
+    // Because of this, we fall back to Velcro default response.
+    let hints
+    try {
+      hints = await this.batchedVelcroDiscovery({ networkId, accountAddr, baseCurrency })
+    } catch (error) {
+      hints = {
+        networkId: networkId,
```

Comment:
we should move this cruft to the type definiitions and just do ` { ...EmptyHints, error }`